public/client-v2/src/components/ErrorBanner.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+import { error } from '../styleConstants';
+import type { ActionError } from '../types/Action';
+
+type Props = {
+  error: ActionError
+};
+
+const ErrorContainer = styled('div')`
+  background-color: ${error.primary};
```

Comment:
I don't think we should change it now but we should probably think about using `styled-components` [themeing](https://www.styled-components.com/docs/advanced#theming) for this. I forgot about it when we talked earlier but it will make it easier to theme things nicely. For now though this can definitely stay as it is!